src/snowflake/snowpark/session.py

```diff
@@ -1286,15 +1290,21 @@ def create_dataframe(
                     f"create or replace temporary table {table_name} ({attribute_to_schema_string(schema._to_attributes())})"
                 )
 
-            t = self.write_pandas(
-                data,
-                table_name,
-                database=sf_database,
-                schema=sf_schema,
-                quote_identifiers=True,
-                auto_create_table=schema is None,
-                table_type="temporary",
-            )
+            try:
+                t = self.write_pandas(
+                    data,
+                    table_name,
+                    database=sf_database,
+                    schema=sf_schema,
+                    quote_identifiers=True,
+                    auto_create_table=schema is None,
+                    # auto_create_table=True,
```

Comment:
let's explicitly set it to False, and don't set `table_type`?

Reply:
Sorry this is some leftover from my local testing. But yeah I think this is a good idea.